tests/test_table.py

```diff
@@ -3288,6 +3294,53 @@ def test_get_table_density(self):
         # Tables with some zeros explicitly defined.
         npt.assert_almost_equal(self.st7.get_table_density(), 0.75)
 
+    def test_to_pandas_pathways(self):
+        dt_rich = Table(np.array([[5, 6, 7], [8, 9, 10], [11, 12, 13]]),
+                        ['1', '2', '3'], ['a', 'b', 'c'],
+                        [{'pathways': [['a', 'bx'], ['a', 'd']]},
+                         {'pathways': [['a', 'bx'], ['a', 'c']]},
+                         {'pathways': [['a']]}],
+                        [{'barcode': 'aatt'},
+                         {'barcode': 'ttgg'},
+                         {'barcode': 'aatt'}])
+        exp_obs = pd.DataFrame(
+            {'pathways': [[['a', 'bx'], ['a', 'd']],
+                          [['a', 'bx'], ['a', 'c']],
+                          [['a']]]}, index=['1', '2', '3'])
+        exp_samp = pd.DataFrame(
+            {'barcode': ['aatt', 'ttgg', 'aatt']},
+            index=['a', 'b', 'c'])
+        exp_counts = pd.SparseDataFrame(
+            np.array([[5., 6., 7.], [8., 9., 10.], [11., 12., 13.]]),
+            index=['1', '2', '3'],
+            columns=['a', 'b', 'c'])
+
+        res_counts, res_samp, res_obs = dt_rich.to_pandas()
+        pdt.assert_frame_equal(res_counts, exp_counts)
+        pdt.assert_frame_equal(res_samp, exp_samp)
+        pdt.assert_frame_equal(res_obs, exp_obs)
+
+    @npt.dec.skipif(HAVE_PD is False, msg='pandas is not installed')
+    def test_to_pandas_taxonomy(self):
+
+        st_rich = Table(self.vals,
+                        ['1', '2'], ['a', 'b'],
+                        [{'taxonomy': ['k__a', 'p__b']},
+                         {'taxonomy': ['k__a', 'p__c']}],
+                        [{'barcode': 'aatt'}, {'barcode': 'ttgg'}])
+
+        exp_counts = pd.SparseDataFrame([[5., 6.], [7., 8.]],
+                                        index=['1', '2'],
+                                        columns=['a', 'b'])
+        exp_obs = pd.DataFrame({'taxonomy': [['k__a', 'p__b'],
+                                              ['k__a', 'p__c']]},
+                               index=['1', '2'])
+        exp_samp = pd.DataFrame({'barcode': ['aatt',  'ttgg']},
+                                index=['a', 'b'])
+        res_counts, res_samp, res_obs = st_rich.to_pandas()
+        pdt.assert_frame_equal(res_counts, exp_counts)
+        pdt.assert_frame_equal(res_samp, exp_samp)
+        pdt.assert_frame_equal(res_obs, exp_obs)
 
```

Comment:
can you add a test please for when an axis has multiple pieces of metadata associated?